arbnode/node.go

```diff
@@ -210,6 +211,10 @@ func ConfigDefaultL1NonSequencerTest() *Config {
 	config.Staker.Enable = false
 	config.BlockValidator.ValidationServerConfigs = []rpcclient.ClientConfig{{URL: ""}}
 	config.Bold.MinimumGapToParentAssertion = 0
+	// Disable broadcast during sync only for path tests as it causes test timeouts
+	if env.GetTestStateScheme() == rawdb.PathScheme {
```

Comment:
Why does that happen?
Also it is possible for a test to use pathdb even if env.GetTestStateScheme() != rawdb.PathScheme.
see builder.RequireScheme